### PR TITLE
home work sending test

### PR DESCRIPTION
**Pull request by Giti-mafakheri**
The home work was not difficult for me, I didn't have enough time to complete all tasks but I am sure that I completely understood the concepts. I just wanted to send it to make sure that every thing is ok in the procedure.

### CODE REVIEW

**Comment by Giti-mafakheri** [comment]
How difficult did you find this (out of 10)? 
2
Was there anything that you struggled with?
No
Is there anything that you'd like some further information on?
No
Roughly how long did it take?
3

File: Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// # Array and Functions Bonus Material
+//
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript.
+//You'll have to remember your pre-work, or do some googling to figure this out.
+
+const maxOfTwoNumbers = function(num1,num2){
```
Comment (CamillaCdC) [inline Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js:6]: Technically this is correct. However for readability we'd prefer you to write it as: 
`const maxOfTwoNumbers = function (num1,num2) {`

File: Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+// 4. Define a function `sumArray` and a function `multiplyArray` that sums and multiplies (respectively) all
+// the numbers in an array of numbers. For example, `sumArray([1,2,3,4])` should return 10,
+// and `multiplyArray([1,2,3,4])` should return 24.
+//
+const sumArray = function(){
+  var array = [1,2,3,4,5];
+  let sum=0 ;
+  for (var i = 0; i < array.length; i++) {
+sum  =sum + array[i];
```
Comment (CamillaCdC) [inline Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js:48]: indentation and spacing

File: Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+
+
+const multiplyArray = function(){
+  var array = [1,2,3,4,5];
+  let mul = 1 ;
+  for (var i = 0; i < array.length; i++) {
+mul  =mul * array[i];
```
Comment (CamillaCdC) [inline Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js:61]: same as above

**Comment by CamillaCdC** [comment]
Glad you found the homework easy!

There is an issue with this pull request, but from my end of things not yours! Going forward you shouldn't have an issue - so don't worry - I'm just going to close this one. Your homework will still be in the repo, I just had to merge it another way to avoid a conflict with a hidden file.

**Pull request closed without merging**